perun-core/src/main/resources/postgresChangelog.txt

```diff
@@ -6,6 +6,11 @@
 -- Directly under version number should be version commands. They will be executed in the order they are written here.
 -- Comments are prefixed with -- and can be written only between version blocks, that means not in the lines with commands. They have to be at the start of the line.
 
+3.1.87
+ALTER TABLE groups_resources ADD COLUMN auto_assign_subgroups boolean default false not null;
+ALTER TABLE groups_resources_automatic ADD COLUMN auto_assign_subgroups boolean default false not null;
```

Comment:
default value for `groups_resources_automatic` could be true as it is set to it every time